flake.nix

```diff
@@ -721,7 +729,10 @@
               '';
           }
         )
-        // { default = self.devShells.${system}.stdenv; }
+        // {
+          default = self.devShells.${system}.stdenv;
+          python = self.packages.${system}.nix.python-bindings.shell;
```

Comment:
It would be nice to have a `python/flake.nix`, so one could just `cd python; nix develop`, but that doesn't work since the main Nix flake and the Python bindings one would depend on each other, and the subflake updating doesn't work very well.

I could create a `shell.nix` so that `cd python; nix-shell` would work (using `builtins.getFlake` on the parent), let me know if that's better.

Reply:
Yeah, it's not independent, so not a subflake. Either subflakes could be made to be more useful, or we design devshells so that they can be declared for a specific subdirectory. While we don't have that, I think a `shell.nix` is nice.